### PR TITLE
Fixed bug in WRONG_DECLARATIONS_ORDER rule

### LINKED ISSUE #1865: Bug in `WRONG_DECLARATIONS_ORDER` rule
## Describe the bug
Looks like it happens due to `private const val DELAY = 15000L // 15s` line with comment.

## Expected behavior
Same but without newline at 7 line

```kotlin
package com.saveourtool.diktat

class ActiveBinsMetric {
    companion object {
        private const val ACTIVE_BINS_METRIC_NAME = "c.concurrent.bins"
        private const val ALL_ACTIVE_BINS_LABEL = "total"
        private const val DELAY = 15000L  // 15s
        private const val EGG_1_BUCKET_LABEL = "1"
        private const val EGG_2_BUCKET_LABEL = "2"
        private const val EGG_3_BUCKET_LABEL = "3"
        private const val EGG_4_5_BUCKET_LABEL = "4-5"
        private const val EGG_7_9_BUCKET_LABEL = "7-9"
        private const val EGG_OVER_10_BUCKET_LABEL = "10+"
        private const val NUMBER_OF_EGGS_LABEL = "numberOfEggs"
        private val numberOfEggsBuckets = setOf(
            EGG_4_5_BUCKET_LABEL,
            EGG_2_BUCKET_LABEL,
            EGG_3_BUCKET_LABEL,
            EGG_7_9_BUCKET_LABEL,
            EGG_1_BUCKET_LABEL,
            EGG_OVER_10_BUCKET_LABEL,
            ALL_ACTIVE_BINS_LABEL)
    }
}
```

## Observed behavior
```kotlin
package com.saveourtool.diktat

class ActiveBinsMetric {
    companion object {
        private const val ACTIVE_BINS_METRIC_NAME = "c.concurrent.bins"
        private const val ALL_ACTIVE_BINS_LABEL = "total"

        private const val DELAY = 15000L  // 15s
        private const val EGG_1_BUCKET_LABEL = "1"
        private const val EGG_2_BUCKET_LABEL = "2"
        private const val EGG_3_BUCKET_LABEL = "3"
        private const val EGG_4_5_BUCKET_LABEL = "4-5"
        private const val EGG_7_9_BUCKET_LABEL = "7-9"
        private const val EGG_OVER_10_BUCKET_LABEL = "10+"
        private const val NUMBER_OF_EGGS_LABEL = "numberOfEggs"
        private val numberOfEggsBuckets = setOf(
            EGG_4_5_BUCKET_LABEL,
            EGG_2_BUCKET_LABEL,
            EGG_3_BUCKET_LABEL,
            EGG_7_9_BUCKET_LABEL,
            EGG_1_BUCKET_LABEL,
            EGG_OVER_10_BUCKET_LABEL,
            ALL_ACTIVE_BINS_LABEL)
    }
}
```

## Steps to Reproduce
```kotlin
class ActiveBinsMetric() {
    companion object {
        private const val EGG_4_5_BUCKET_LABEL = "4-5"
        private const val EGG_3_BUCKET_LABEL = "3"
        private const val EGG_OVER_10_BUCKET_LABEL = "10+"
        private const val EGG_7_9_BUCKET_LABEL = "7-9"
        private const val DELAY = 15000L // 15s
        private const val ACTIVE_BINS_METRIC_NAME = "c.concurrent.bins"
        private const val NUMBER_OF_EGGS_LABEL = "numberOfEggs"
        private const val ALL_ACTIVE_BINS_LABEL = "total"
        private const val EGG_2_BUCKET_LABEL = "2"
        private const val EGG_1_BUCKET_LABEL = "1"
        private val numberOfEggsBuckets = setOf(
            EGG_4_5_BUCKET_LABEL,
            EGG_2_BUCKET_LABEL,
            EGG_3_BUCKET_LABEL,
            EGG_7_9_BUCKET_LABEL,
            EGG_1_BUCKET_LABEL,
            EGG_OVER_10_BUCKET_LABEL,
            ALL_ACTIVE_BINS_LABEL)
    }
}
```

### PR DESCRIPTION
**Pull request by diphtongue**
### What's done:
- Fixed new line, which appeared before line with comment.
- Added smoke test.

Closes #1865

### CODE REVIEW

**Review by DrAlexD — APPROVED** [review]

File: diktat-rules/src/test/kotlin/com/saveourtool/diktat/ruleset/chapter3/ClassLikeStructuresOrderFixTest.kt [hunk 1]
```diff
@@ -32,4 +32,10 @@ class ClassLikeStructuresOrderFixTest : FixTestBase("test/paragraph3/file_struct
     fun `regression - should not move loggers that depend on other variables from scope`() {
         fixAndCompare("LoggerOrderExpected.kt", "LoggerOrderTest.kt")
     }
+
+    @Test
+    @Tags(Tag(WarningNames.BLANK_LINE_BETWEEN_PROPERTIES))
```
Comment (nulls) [inline diktat-rules/src/test/kotlin/com/saveourtool/diktat/ruleset/chapter3/ClassLikeStructuresOrderFixTest.kt:37]:
```suggestion
    @Tag(WarningNames.BLANK_LINE_BETWEEN_PROPERTIES)
```

**Review by nulls — APPROVED** [review]

File: diktat-rules/src/test/kotlin/com/saveourtool/diktat/ruleset/chapter3/ClassLikeStructuresOrderRuleWarnTest.kt [hunk 2]
```diff
@@ -196,6 +196,37 @@ class ClassLikeStructuresOrderRuleWarnTest : LintTestBase(::ClassLikeStructuresO
...
     }
 
+    @Test
+    @Tag(WarningNames.BLANK_LINE_BETWEEN_PROPERTIES)
+    fun `should not trigger on EOL comment on the same line with property`() {
+        lintMethod(
+            """
+                    |class ActiveBinsMetric(meterRegistry: MeterRegistry, private val binRepository: BinRepository) {
+                    |    companion object {
+                    |        private const val EGG_4_5_BUCKET_LABEL = "4-5"
```
Comment (nulls) [inline diktat-rules/src/test/kotlin/com/saveourtool/diktat/ruleset/chapter3/ClassLikeStructuresOrderRuleWarnTest.kt:206]: I'm not sure that test requires so many lines. can be simplified to 1-2 fields

Comment (diphtongue) [reply diktat-rules/src/test/kotlin/com/saveourtool/diktat/ruleset/chapter3/ClassLikeStructuresOrderRuleWarnTest.kt]: Done

File: diktat-rules/src/test/kotlin/com/saveourtool/diktat/ruleset/chapter3/ClassLikeStructuresOrderFixTest.kt (hunk 1 above)
Comment (diphtongue) [reply diktat-rules/src/test/kotlin/com/saveourtool/diktat/ruleset/chapter3/ClassLikeStructuresOrderFixTest.kt]: Done

**Pull request merged**